odyssey/templates/deployment-odyssey.yaml

```diff
@@ -21,11 +24,6 @@ spec:
         {{- with .Values.additionalAnnotations }}
           {{- toYaml . | nindent 8 }}
         {{- end }}
-      labels:
-        {{- include "odyssey.selectorLabels" . | nindent 8 }}
```

Comment:
This part has not been moved like the other duplicate parts. Is it wanted ?

Reply:
yes "selectorLabels" are included in "labels"